nixos/modules/services/network-filesystems/litestream.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.litestream;
+in
+{
+  options.services.litestream = {
+    enable = mkEnableOption "litestream";
+
+    package = mkOption {
+      description = "Package to use.";
+      default = pkgs.litestream;
+      defaultText = "pkgs.litestream";
+      type = types.package;
+    };
+
+    config = mkOption {
```

Comment:
Could we make this a `nix` attribute set? See https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md for motivation.

```
settings = mkOption {
  type = (pkgs.format.yaml {}).format;
  ...
};
```

Reply:
Fixed

Reply:
Nice! :rocket: